apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx

```diff
@@ -85,6 +86,10 @@ const getAddressChangePostfix = (sectionName, sectionType, floorName, unitName,
     return addressChangePostfix
 }
 
+const isDeclineTicketChangeByResident = (ticketChange) =>
```

Comment:
Maybe you can find a better naming for that? IMO it's long a bit)
```suggestion
const isDeclinedByResident = (ticketChange) =>
```